scripts/default.nix

```diff
@@ -0,0 +1,10 @@
+{pkgs}: let
+  sa = pkgs.writeShellApplication;
+  format = sa {
+    name = "format";
+    text = builtins.readFile ./format.sh;
+    runtimeInputs = with pkgs; [alejandra nix-linter];
+  };
```

Comment:
I know this seems like a lot for a trival script where I could probably just use `writeShellScriptBin` but this approach scales better as more scripts are added.